### DISCUSSION TITLE
Access to the group

### DISCUSSION THREAD

**Discussion by fidele007** [Rim-El-Ballouli/entretien-naturalisation-francaise#3] [Q&A] [answered]
Hi,
I would like to get access to the group. I've requested a few days ago and still haven't gotten any reply.

Thanks!

**Comment by Rim-El-Ballouli** [comment, marked as answer]
Hello, 

@fidele007  The access has been granted :). Good luck!

**Reply by fidele007** [reply]
Thanks! ^^
